keepalived/check/check_daemon.c

```diff
@@ -217,6 +219,7 @@ reload_check_thread(__attribute__((unused)) thread_t * thread)
 #ifdef _WITH_VRRP_
 	kernel_netlink_close();
 #endif
+	script_killall(master, SIGHUP);
```

Comment:
I chose a SIGHUP when reload just in case.
However, we want that it terminate. So, it looks good in SIGTERM.

I thought as a child process is there is no need to understand the cause of SIGTERM. And I think so child process (as the script program) would be required to terminate with SIGTERM.